.github/workflows/main.yml

```diff
@@ -0,0 +1,39 @@
+
+name: CI
+
+# Run tests on pull requests and when changes are directly
+# commited to master.
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  # Run static/code-quality checks
+  check:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/setup-go@v2-beta
```

Comment:
are this actions from github itself? Because I don't see them in this PR.

Reply:
Yes, there's an implicit "https://github.com/" prefix. If you put it there yourself you get https://github.com/actions/setup-go which is a valid repo hosted on github. You can host your own this way too. So if I created "https://github.com/phlogistonjohn/foobar" you could reference it using phlogistonjohn/foobar (note that is not a valid link :-)).